openfl/component/director/director.py

```diff
@@ -34,24 +260,29 @@ def __init__(
             settings: dict = None
     ) -> None:
         """Initialize a director object."""
-        # TODO: add working directory
-        super().__init__()
         self.sample_shape, self.target_shape = sample_shape, target_shape
         self._shard_registry = {}
-        self.col_exp_queues = defaultdict(asyncio.Queue)
-
-        self.experiment_data = {}  # {Experiment name : archive bytes}
-        # What if two experiments come with the same name from different users?
-        self.experiments_queue = asyncio.Queue()  # experiments waiting to be executed
-        self.experiment_stash = defaultdict(dict)  # Running of finished experiments
-        # {API name : {experiment name : aggregator}}
+        self.experiments = ExperimentsList()
 
         self.tls = tls
         self.root_certificate = root_certificate
         self.private_key = private_key
         self.certificate = certificate
         self.settings = settings or {}
 
+    async def start(self):
```

Comment:
Looks like it is not `director.start`. Director will start without this method. It's a background task, that director executes.